### ISSUE TITLE
Godot 4

### ISSUE THREAD

**Issue by revpriest** [Cykyrios/Godot360#2]
Thanks for this camera system, looks like it'd do what I need, but it seems like it doesn't work with Godot 4.

Any plans to do so? Whatever conversion the machine tried didn't help and the chatbot couldn't handle it either. I don't really know any gdsript currently, working with csharp.

**Comment by Cykyrios** [comment]
Hi, I'll port what I can to Godot 4 over the weekend, I know fisheye works because I already use a modified version of it in another Godot 4 project.

**Comment by Cykyrios** [comment]
Actually I ported the project this evening, the latest commit should be good to go.
Two things to note:
- I increased the FoV of each camera to 100° instead of 90, in part to decrease artifacts from screen space effects, and in part because in Godot 4, there are visible seams between the cameras otherwise.
- All sub-cameras should have auto-exposure disabled to prevent visible brightness inconsistencies.
